src/index.ts

```diff
@@ -14,10 +14,13 @@ import {
 } from '@jupyterlab/notebook';
 import { Cell } from '@jupyterlab/cells';
 import { MySTContentFactory } from './MySTContentFactory';
+import { IBibliographyManager, BibliographyManager } from './bibliography';
+
 import { IRenderMimeRegistry } from '@jupyterlab/rendermime';
 
 import { notebookCellExecuted } from './actions';
 import { mystMarkdownRendererFactory } from './mime';
+import { citationRenderers } from './myst';
```

Comment:
@agoose77 we need some way to share the citation renderer into the various places we use those renderer. I don't think they necessarily need to be "connected" (i.e. an update to a bib file doesn't really need to force a re-render, but the next render should bring in the right citation).

The way we have done this in the CLI is to have a state manager, that is indexed based on file name, and then we combine citation renderers before we render the markdown.

I think this could work, and maybe that works? Curious if you know of a better way to share state between the plugins?